### PR TITLE
Switch to scylladb/gocql

### PR DESCRIPTION
**Pull request by dkropachev**
Switch from [github.com/gocql/gocql](https://github.com/gocql/gocql) to [github.com/scylladb/gocql](https://github.com/scylladb/gocql)

### CODE REVIEW

File: cmd/schemagen/map_types.go [hunk 1]
```diff
@@ -88,18 +85,3 @@ func mapScyllaToGoType(s string) string {
...
-	case "gocql.NativeType":
-		return t.(gocql.NativeType).String()
-	case "gocql.CollectionType":
-		collectionType := t.(gocql.CollectionType).String()
-		collectionType = strings.Replace(collectionType, "(", "<", -1)
-		collectionType = strings.Replace(collectionType, ")", ">", -1)
-		return collectionType
-	default:
-		panic(fmt.Sprintf("Did not expect %v type in user defined type", tType))
-	}
```
Comment (sylwiaszunejko) [inline cmd/schemagen/map_types.go:104]: why remove this function?

Comment (dkropachev) [reply cmd/schemagen/map_types.go:104]: There is difference between `KeyspaceMetadata` in  [scylladb/gocql](https://github.com/scylladb/gocql/blob/c570ec02e0641060c3ebedfe09703977dd6ed8b3/metadata.go#L48-L59) and [gocql/gocql](https://github.com/gocql/gocql/blob/c570ec02e0641060c3ebedfe09703977dd6ed8b3/metadata.go#L48-L59)

On `scylladb/gocql` attribute `Type` is string, a raw name of the type, while on `gocql/gocql` it is `TypeInfo`.
So, this function converts `TypeInfo` to a string, a raw name of the type.
Which is why this function is not needed anymore

**Comment by sylwiaszunejko** [comment]
@dkropachev pls rebase to master

**Review by sylwiaszunejko — APPROVED** [review]

**Comment by sylwiaszunejko** [comment]
@mykaul maybe you want to look at it before I merge it?

**Comment by mykaul** [comment]
No, I'm good. I'm a bit surprised about the number of dependencies updated, but I guess it is needed - did not check it (I did verify dependentbot is enabled for this repo)

**Pull request merged**